contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -632,8 +628,6 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
         // DELEGATECALL does not change anything about the message context.
         MessageContext memory nextMessageContext = messageContext;
         
-        bool isStaticEntrypoint = false;
```

Comment:
@ben-chain was this recently made obsolete?

Reply:
Oooh I recognize this variable!  This was from when we thought nested `ovmCALLs` within an `ovmSTATICCALL` should all collapse up atomically if they attempted a violation.  It's been obsolete for a while, actually; surprised to see it in here still--makes sense to me that it would've been an `unused` error.  Yay code removal! :D